GLib-2.0.gir

```diff
@@ -42421,6 +42495,11 @@ will not reflect any change once the special directories are loaded.</doc>
         </parameter>
       </parameters>
     </function>
+    <function name="get_user_state_dir" c:identifier="g_get_user_state_dir">
```

Comment:
This also has a since annotation upstream

Reply:
Indeed: https://gitlab.gnome.org/GNOME/glib/-/commit/68eab1d99981a8cc6865239500fe4a84878d880c#cdd7af8df16d98cc412ce15aec4bd635d70e8450_1973_2022

Did you report this somewhere?

Reply:
opened an issue at https://gitlab.gnome.org/GNOME/gobject-introspection/-/issues/412